lib/helpers/normalizeArgs.js

```diff
@@ -2,8 +2,8 @@
 
 var assert = require('assert');
 
-var map = require('lodash.map');
-var flatten = require('lodash.flatten');
+var map = require('collection-map');
```

Comment:
I think this should be `arr-map`.  It's fine adding the extra dependency because it should be deduped with `collection-map`